terminus/geometry/latlon.py

```diff
@@ -38,15 +38,22 @@ def __eq__(self, other):
         if self.lat != other.lat:
             return False
         else:
-            return (self.lon == other.lon) or (abs(self.lon) == 180 and abs(other.lon) == 180) or abs(self.lat) == 90
+            return (self.lon == other.lon) or \
+                   (abs(self.lon) == 180 and abs(other.lon) == 180) or \
+                abs(self.lat) == 90
```

Comment:
Not sure from the top of my head what will pep8 tell, but can you align this last line with the ones above?